recipes/xorg-libxp/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set xorg_name = "libXp" %}
+{% set xorg_category = "lib" %}
+{% set name = "xorg-" ~ xorg_name %}
+{% set version = "1.0.3" %}
+{% set sha256 = "7f360c9905849c3587d48efc0f0ecbc852c19f61a52b18530d6b005cb9148c57" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.bz2
+  sha256: {{ sha256 }}
+
+build:
+  number: 2
+  detect_binary_files_with_prefix: true
+  # windows?
+  skip: True  # [win]
```

Comment:
Is it confirmed that this package will not build on Windows?

If so, a lot of the build script infrastructure *could* be removed ... but also see an appeal to leaving it in place, for consistency.